src/lib/components/ArticleHeader.svelte

```diff
@@ -77,6 +87,9 @@
       <a href="/{href}" class='flex flex-col space-y-2'>
         <h2 class='text-lg font-bold'>{title}</h2>
         <h3 class='text-base font-normal'>by {author}</h3>
+        {#if version != '1'}
+        <h3 class='text-base font-normal'>version: {version}</h3>
+        {/if}
```

Comment:
```suggestion
        {#if version != '1'}
          <h3 class='text-base font-normal'>version: {version}</h3>
        {/if}
```

Stylistic convention is to indent this line.